### PR TITLE
ADD HomePage and VerifyEmail

### PR DESCRIPTION
**Pull request by Nada14samy**

### CODE REVIEW

**Review by Nada14samy — COMMENTED** [review]
Home page 
Verify Email

**Review by Nada14samy — COMMENTED** [review]
ADD HomePage and VerifyEmail

File: src/sections/SignUp/SignUp.jsx [hunk 1]
```diff
@@ -1,178 +1,119 @@
...
+            label="Repeat Password"
+            register={register}
+            errors={errors}
+            placeholder="Repeat Password..."
+            type="password"
+           
+          />
+
+          <div className="flex justify-center w-full">
+            <button
```
Comment (mohamedTarek21) [inline src/sections/SignUp/SignUp.jsx:101]: Use generic button component

File: src/sections/Footer/Footer.css [hunk 2]
```diff
@@ -1,56 +1,10 @@
-@import "../../index.scss";
```
Comment (mohamedTarek21) [inline src/sections/Footer/Footer.css:1]: delete file if you do not need it or you can use tailwind classes instead

**Review by Nada14samy — COMMENTED** [review]
Change button

**Review by mohamedTarek21 — APPROVED** [review]

**Pull request merged**